src/api/app/models/bs_request.rb

```diff
@@ -52,6 +52,8 @@ class SaveError < APIException
   has_many :request_history_elements, -> { order(:created_at) }, class_name: 'HistoryElement::Request', foreign_key: :op_object_id
   has_many :review_history_elements, through: :reviews, source: :history_elements
 
+  belongs_to :user, foreign_key: 'creator', primary_key: 'login'
```

Comment:
grep says everywhere is 75 places. Minus wrong hits it might be 40 to 50. This is not much, especially since you would just have to replace 'creator' with 'creator.login'.

And what's the point of introducing an association and then not using it?